packages/ledger-amino/src/launchpadledger.ts

```diff
@@ -166,6 +167,17 @@ export class LaunchpadLedger {
     await this.verifyCosmosAppIsOpen();
   }
 
+  public async verifyAddress(hdPath: HdPath): Promise<AddressAndPublicKeyResponse | ErrorResponse> {
```

Comment:
The errors are converted to exceptions, right? So this becomes

```suggestion
  public async verifyAddress(hdPath: HdPath): Promise<AddressAndPublicKeyResponse> {
```